AAP-53971-RN-subtitle-26 (#4348)
ansible/aap-docs: downstream/titles/release-notes/topics/aap-26.adoc

[id="new-features"]
= New features and enhancements

== General availability of {AAPchatbot}

The {AAPchatbot} is now generally available on {PlatformNameShort} 2.6 on {OCPShort}. It is an intuitive chat interface embedded within the {PlatformNameShort}, utilizing generative artificial intelligence (AI) to answer questions about the {PlatformNameShort}.

The chat experience in the {AAPchatbot} interacts with users in their natural language prompts in English, and uses large language models (LLMs) to generate quick, accurate, and personalized responses. These responses empower users to work more efficiently, thereby improving productivity and the overall quality of their work.

To access and use the {AAPchatbot}, you need:
* {PlatformNameShort} 2.6 on {OCPShort}
* An LLM service that is hosted on either {RHELAI} or {OCPAI}

> For more information, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/installing_on_openshift_container_platform/index#deploying-chatbot-operator[Deploying the Ansible Lightspeed intelligent assistant on {OCPShort}] in Installing on {OCPShort}.

Author: Hi @jself-sudoku, which line or part do you mean to verify here?

> For more information, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/installing_on_openshift_container_platform/index#deploying-chatbot-operator[Deploying the Ansible Lightspeed intelligent assistant on {OCPShort}] in Installing on {OCPShort}.

Reviewer: Oh, it's in the link you have there. 
```suggestion
For more information, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/**2.5**/html-single/installing_on_openshift_container_platform/index#deploying-chatbot-operator[Deploying the Ansible Lightspeed intelligent assistant on {OCPShort}] in Installing on {OCPShort}.
```

> For more information, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/installing_on_openshift_container_platform/index#deploying-chatbot-operator[Deploying the Ansible Lightspeed intelligent assistant on {OCPShort}] in Installing on {OCPShort}.

Author: Oh okay perfect, I will check with them. Thank you.

== General availability of Ansible self-service automation portal

Ansible self-service automation portal is now generally available as part of the {PlatformNameShort} subscription. The new self-service automation portal empowers platform admins to provide a streamlined “point-and-click” Ansible automation experience to a broader set of users within the organization. Users who aren’t Ansible experts now have a dedicated self-service portal from which they can launch a range of automation jobs.
* Installation options: Deploy self-service automation portal as a separate installation on {OCP} using a Helm chart. 
* Syncs existing automation content: Extend the reach and impact of your automation job templates, while maintaining full control and compliance. 
* Seamless Integration: Uses your existing {PlatformNameShort} configuration—same logins, same security controls, same automation logic.

> * Simplified Interface: A distinct, user-friendly web interface designed for business users, not automation experts

Reviewer: ```suggestion
* Simplified Interface: A distinct, user-friendly web interface designed for business users, not automation experts.
```

* *New event filter*: A new filter plugin, `event_splitter`, is available to handle and process nested events more effectively.
* *Rulebook concurrency key*: Rulebooks now support a concurrency key, enabling you to group events by resource to ensure they are processed sequentially.

== Installation changes

=== Containerized installation
Updated system requirements for containerized installation of {PlatformNameShort} include: 

* The {RHEL} 9.2 operating system requirement was updated to 9.4 or later minor versions of {RHEL} 9.
* {PostgresVers} 16 and 17 are now supported for customer provided (external) databases. 
+
[NOTE]
====
External databases using PostgreSQL 16 or 17 must rely on external backup and restore processes. Backup and restore functionality is dependent on utilities provided with PostgreSQL 15.
====

For more information see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.6/html/containerized_installation/aap-containerized-installation#system-requirements[System requirements] in _Containerized installation_.

=== Operator installation

> Updated system requirements for {OperatorPlatformNameShort} on Red Hat OpenShift include:

Reviewer: ```suggestion
Updated system requirements for {OperatorPlatformNameShort} on {OCP} include:
```

The {RHEL} 9.2 operating system requirement was updated to 9.4 or later minor versions of {RHEL} 9.
PostgreSQL 16 and 17 are now supported for customer-provided (external) databases. 
+
[NOTE]
====
External databases using PostgreSQL 16 or 17 must rely on external backup and restore processes. Backup and restore functionality is dependent on utilities provided with PostgreSQL 15.
====

For more information about the {OperatorPlatformNameShort} system requirements, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/tested_deployment_models/ocp-topologies[Operator topologies] in _Tested deployment models_. 

=== RPM installation

> Updated system requirements for RPM installation of {PlatformNameShort} 2.6 include:

Reviewer: ```suggestion
Updated system requirements for RPM installation of {PlatformNameShort} {PlatformVers} include:
```

> * {PlatformNameShort} RPM installer was deprecated in 2.5 and will be removed in {PlatformNameShort} 2.7. The RPM installer will be supported for RHEL 9 during the lifecycle of {PlatformNameShort} 2.6 to support migrations to existing supported topologies. See the support matrix for more information on upgrade and migration paths.

Reviewer: ```suggestion
* {PlatformNameShort} RPM installer was deprecated in 2.5 and will be removed in {PlatformNameShort} 2.7. The RPM installer will be supported for RHEL 9 during the lifecycle of {PlatformNameShort} {PlatformVers} to support migrations to existing supported topologies. See the support matrix for more information on upgrade and migration paths.
```
